lib/pure/httpclient.nim

```diff
@@ -425,6 +425,41 @@ proc newMultipartData*: MultipartData =
   ## Constructs a new ``MultipartData`` object.
   MultipartData(content: @[])
 
+proc parseMultipartDataString(item: string): string =
+  var itemPart: seq[string] = item.
```

Comment:
Splitting the line on an ending dot (period) like this I think is not used on the Style of Nim.
Maybe run `nimpretty` on it and then adjust it manually :grey_question: 
:thinking:

Reply:
nimpretty breaks it after `(`